langtrace/langtrace-elastic-demo.py

```diff
@@ -0,0 +1,84 @@
+import os
+import asyncio
+from dotenv import load_dotenv
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from langtrace_python_sdk import langtrace, with_langtrace_root_span
+from langchain_openai import AzureChatOpenAI
+from langchain.prompts import ChatPromptTemplate
+from langchain.schema.runnable import RunnableSequence
+from langchain_community.tools import DuckDuckGoSearchResults
+from langchain_community.utilities import DuckDuckGoSearchAPIWrapper
+
+# Load environment variables
+load_dotenv()
+
+# Elastic APM settings
+ELASTIC_APM_SERVER_URL = os.getenv("ELASTIC_APM_SERVER_URL")
+SERVICE_NAME = os.getenv("SERVICE_NAME", "langtrace-elastic-demo")
+ELASTIC_APM_SECRET_TOKEN = os.getenv("ELASTIC_APM_SECRET_TOKEN")
+
+# Set up Elastic exporter
+elastic_exporter = OTLPSpanExporter(
+    endpoint=ELASTIC_APM_SERVER_URL,
+    headers={"Authorization": f"Bearer {ELASTIC_APM_SECRET_TOKEN}"}
+)
+
```

Comment:
So I wanted to see how to use auto-instrument. Hence do we need to set this here vs leave it set outside of the code?

Reply:
setting the exporter and the auth code.

Reply:
Yes you can set this as ENV vars outside of the code.